index.html

```diff
@@ -0,0 +1,1793 @@
+                </g>
+            </svg>
+            <strong class="mt-5 sinp">SIGNUP</strong>
+            <div class="bg-white px-2 subs-div mt-5">
+                <strong class="subscribe">SUBSCRIBE <i class=" px-0 fa fa-arrow-right" aria-hidden="true"></i></strong>
+            </div>
+            <div class="ava-nav navbar  d-lg-none  navbar-dark">
+                <button class="navbar-toggler d-lg-none" type="button" data-toggle="collapse" data-target="">
+                    <span class="navbar-toggler-icon"></span>
+                </button>
+            </div>
+        </nav>
+        <div class="hidnav">
+            <div class="allnav">
+                <ul class="nav ul-v bg-white navbar-black text-uppercase d-flex justify-content-between">
+                    <li class="nav-item liv"><a class="nav-link font-weight-bold" href="">usa</a></li>
+                    <li class="nav-item liv"><a class="nav-link font-weight-bold" href="">london</a></li>
+                    <li class="nav-item liv"><a class="nav-link font-weight-bold" href="">advert</a></li>
+                    <li class="nav-item liv"><a class="nav-link font-weight-bold" href="">sport</a></li>
+                    <li class="nav-item liv"><a class="nav-link font-weight-bold" href="">busines</a></li>
+                    <li class="nav-item liv"><a class="nav-link font-weight-bold" href="">tech & science</a></li>
+                    <li class="nav-item liv"><a class="nav-link font-weight-bold" href="">organiser</a></li>
+                    <li class="nav-item liv"><a class="nav-link font-weight-bold" href="">culture</a></li>
+                    <li class="nav-item liv"><a class="nav-link font-weight-bold" href="">culture</a></li>
+                    <li class="nav-item liv">
+                            <form>
+                                <input class="font-weight-bold" type="text" name="search" placeholder="Search..">
+                            </form>
+                    </li>
+                </ul>
+            </div>
+        </div>
+
+
+
+        <!-- BODY FIRST SECTION -->
+        <div class="ff-main mt-3">
+            <div class="m-2 ff-mini ff-mini1">
+                <small class="my-0">FEATURE STORIES</small>
```

Comment:
Update the title of each section to match the ones from the original page.

Reply:
thanks for the changes you requested for better look please i am requesting for another code review please help me and approve me.